### PR TITLE
Assignments

### PR DESCRIPTION
**Pull request by codinglikeagirl42**
To Check Assignments 2, 3 and 4 and Instruction for Using Github w/Jupyter Notebooks.

### CODE REVIEW

**Review by gildedgardenia — CHANGES_REQUESTED** [review]
Overall, I think you did a good job with this! When reviewing it, I noticed some grammatical errors. Other than that, I have some suggestions that might ease readability for some of our students.

1. Break Assignment 4 into multiple pages with one page per checkpoint. We have done this with larger assignments and it can really students understand how the assignment breaks down into separate pieces.
2. I suggest you review your alt text. If I closed my eyes and someone read it to me, I would not be able to put together what the image is about. Descriptors like "a screenshot" can be removed entirely. In your alt text, you also used the phrase "what you'll see" which considering the target audience for alt text is visually impaired users is a poor choice of words.

File: src/assignments/call-center-cleanup.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  customer waits for representative to pick up a call. This is tracked in seconds.
+* During Call Wait Time - amount of time during call that customer has to wait for 
+  representative
+
+Getting Started
+---------------
+
+Fork `this GitHub repository <https://github.com/gildedgardenia/call-center-cleanup-assignment>`__
+and clone it to your computer. If you need a refresher on how to do this see 
+:ref:`Instruction for Using Github w/Jupyter Notebooks<usingGitHubNotebooks>`. Follow the
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/assignments/call-center-cleanup.rst:65]: Did this not build correctly for you?  The syntax for :ref:`link to a different section<Cross-References to Locations in the Same Document>`. There isn't suppose to be a space.

File: src/appendices/jupyter-notebooks/using-github-notebooks.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  * - ls
+    - Ran ls to list contents.
+  * - cd Codergirl
+    - Switch to Codergirl folder.
+  * - git clone <github url> 
+    - (paste link you copied earlier)
+  * - ls
+    - Ran ls to see if my project cloned
+  * - cd EDAExercises
+    - Switch into my new project folder
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/appendices/jupyter-notebooks/using-github-notebooks.rst]: I've added syntax highlighting and changed Codergirl to Women+ in both terminal images.

**Comment by codinglikeagirl42** [comment]
Thank you Sally for the review and great insights.  I've fixed the alt text to better reflect the images. 

I originally had Assignment 4 broken down into individual pages for each checkpoint but did not like how it flowed.  I prefer it on on page and have created links at the very top that would take you to each checkpoint.  There are links after each checkpoint that take you back to the top.  This helps keep the idea of one cohesive final project.

**Comment by codinglikeagirl42** [comment]
Separated out final project into individual pages.  Will add links to samples in different PR

**Pull request merged**